src/app/inbox/components/notification-header/notification-header.component.ts

```diff
@@ -0,0 +1,118 @@
+import { Component, ElementRef, HostBinding, Inject, Input, OnInit, ViewChild } from '@angular/core'
+import { InboxNotification, InboxNotificationAmended } from '../../../types/notifications.endpoint'
+import { WINDOW } from '../../../cdk/window'
+import { environment } from '../../../../environments/environment.local'
+import { UserService } from '../../../core'
+import { take } from 'rxjs/operators'
+import { uiNotificationType } from '../../../types/notifications.local'
+
+@Component({
+  selector: 'app-notification-header',
+  templateUrl: './notification-header.component.html',
+  styleUrls: [
+    './notification-header.component.scss',
+    './notification-header.component.scss-theme.scss'
+    ],
+  preserveWhitespaces: true
+})
+export class NotificationHeaderComponent implements OnInit {
+  @Input() notification: InboxNotificationAmended
+  @ViewChild('header') header: ElementRef<HTMLElement>
+  @HostBinding('class.archived') _archived = false
+  @HostBinding('class.green') green = false
+  @HostBinding('class.orange') orange = true
+  @HostBinding('class.blue') blue = false
+  notificationType: uiNotificationType
+  baseUri: string
+  displayName: string
+  orcidId: string
+  title: string
+
+  constructor(
+    @Inject(WINDOW) private window: Window,
+    _userInfo: UserService,
+  ) {
+    _userInfo
+      .getUserSession()
+      .pipe(take(1))
+      .subscribe((userInfo) => {
+        this.displayName = userInfo.displayName
+        this.orcidId = userInfo.userInfo.REAL_USER_ORCID
+      })
+
+    this.baseUri = 'https:' + environment.BASE_URL
+  }
+
+  ngOnInit(): void {
+    this.title = this.notificationTitle(this.notification)
+    this.notificationType = this.getNotificationType(this.notification)
+    this.setNotificationColor(this.notificationType)
+  }
```

Comment:
Please do not duplicate the code for the following methods, pass this as parameters from the notification.component